### PR TITLE
refactor: validate intersection in S3 when updating metadata

### PR DESCRIPTION
**Pull request by heliprokupetz14**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                       |
| Breaking change | ✔                                                                       |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            |   ✔                                                                     |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,15 +59,16 @@
...
     "@turf/area": "^6.5.0",
     "@turf/bbox-polygon": "^6.5.0",
     "@turf/ellipse": "^6.5.0",
     "@turf/intersect": "^6.5.0",
     "@turf/union": "^6.5.0",
     "@types/geojson": "^7946.0.8",
     "ajv": "^8.12.0",
+    "aws-sdk": "^2.1555.0",
     "axios": "^0.27.2",
+    "compression": "^1.7.4",
```
Comment (netanelC) [inline package.json:71]: Why did you move lines?
Refactor this file so I will see minimum changes

**Review by netanelC — CHANGES_REQUESTED** [review]
This is not how you make a PR.
If the PR is not ready, make this a draft PR

File: package.json (hunk 1 above)
Comment (heliprokupetz14) [reply package.json:71]: refactored

**Review by netanelC — APPROVED** [review]
thank you for your help :) and sorry about for the delay in the task.

**Pull request merged**